src/main/java/de/zalando/aruha/nakadi/repository/kafka/KafkaTopicRepository.java

```diff
@@ -162,8 +162,7 @@ else if (Cursor.BEFORE_OLDEST_OFFSET.equals(pInfo.getNewestAvailableOffset())) {
                         .orElse(false));
     }
 
-    @Override
-    public void postEvent(final String topicId, final String partitionId, final String payload) throws ServiceUnavailableException {
+    void postEvent(final String topicId, final String partitionId, final String payload) throws ServiceUnavailableException {
```

Comment:
I forgot to remove this method. I know it's out of the scope of this task, but if you want to remove it, feel free.